apps/bfd-data-fda/src/main/java/gov/cms/bfd/data/fda/lookup/FdaDrugCodeDisplayLookup.java

```diff
@@ -167,12 +113,12 @@ protected Map<String, String> readFDADrugCodeFile(
    *
    * <p>See {@link gov.cms.bfd.data.fda.utility.App} for details.
    *
-   * @param includeFakeDrugCode is whether to incliude fake drug code
+   * <p>// * @param includeFakeDrugCode is whether to incliude fake drug code
```

Comment:
I think we can remove this line since this param is gone

Reply:
removed